app/routes/report/image/index.js

```diff
@@ -128,31 +128,23 @@ router.route('/')
 // Route for getting an image
 router.route('/retrieve/:key')
   .get(async (req, res) => {
-    let keys = [];
-
-    if (!req.params.key.includes(',')) {
-      keys.push(req.params.key);
-    } else {
-      keys = req.params.key.split(',');
-    }
+    const keys = (req.params.key.includes(',')) ? req.params.key.split(',') : [req.params.key];
```

Comment:
this isn't different logic at all right, just tidier?

Reply:
Correct